src/client.ts

```diff
@@ -193,4 +198,13 @@ export class AutoblocksAPIClient {
   }): Promise<DatasetWithItems> {
     return this.get(`/datasets/${args.datasetId}`);
   }
+
+  public async getTestCases(args: {
+    testSuiteId: string;
+  }): Promise<ManagedTestCase[]> {
+    const result: { testCases: ManagedTestCase[] } = await this.get(
+      `/test-suites/${args.testSuiteId}/test-cases`,
+    );
+    return result.testCases;
```

Comment:
for all of the other methods on this class we return the response as-is to maintain parity. this also means this will only ever be able to expose the `testCases` field and if we ever updated the endpoint to return more fields, e.g. to support pagination, we would need to make a breaking change in order to expose the other fields (like `page` or `nextPage` or w/e)

should we update the response type to be `Promise<{ testCases: ManagedTestCase[] }` so that it matches the response type of the REST API? and usage would be:

```ts
const { testCases } = await client.getTestCases(...);
```

this would be consistent w/ how the current `getDataset` method works as well (returns an object w/ an `items` key)

wdyt @adamnolte ?

Reply:
yes lets always return an object instead of an array so its easier to change

Reply:
I was going back and forth on this as well. 

Main thought that lead me here is you can't do nice de-structured on the python side (I am doing the same pattern there) and I wanted the implementations to be ergonomic in both 

I could go either way - whatever you think reads better 😃

Reply:
let's do an object so its easier to change

Reply:
Updated 👍